data/transport missions.txt

```diff
@@ -1412,6 +1415,138 @@ mission "Hallucination"
 
 
 
+mission "Shady passenger transport 1"
+	name "Shady passenger transport"
+	description "Discreetly transport a passenger to <destination>. If you are caught with him, you may be fined."
+	minor
+	repeat
+	source
+		attributes "near earth" "core" "urban"
+	destination
+		attributes "frontier" "dirt belt" "rim" "north" "south" "pirate"
+		distance 2 3
+	passengers 1
+	illegal 75000 `Your passenger turns out to be a small-time wanted criminal. As he is hauled off in chains, you vehemently protest that you knew nothing of his shady history. The authorities threaten you with jail time, and you quickly contact a lawyer. You wind up quite a bit poorer through a combination of fines, bribes, and attorney's fees (the breakdown remains somewhat unclear).`
```

Comment:
I'd change these `illegal` messages to be more like the current ones for the pirate passenger jobs; the passenger gets hauled off, you might protest a bit, but then you're only given a fine or have to bribe your way out. No need to drag lawyers and attorney fees into this.